doc/BEGINNERS_TUTORIAL.org

```diff
@@ -0,0 +1,184 @@
+* Why Spacemacs?
+- Unparallelled text and structure editing for all types of writing tasks: creative writing, blogging, note-taking, todo-lists,
+  scientific papers... 
+- Powerful modes for programming in dozens of programming languages
+- Deeply customizable yet beginner-friendly
+* Installation
+ Spacemacs is a beginner-friendly and powerful distribution of a popular text
```

Comment:
> Spacemacs is ... distribution

I wonder if we should call it 'distribution'. Because people often confuse Spacemacs with replacement for Emacs, while it's actually extension, a set of configurations. But I am not native English speaker, so that might be my problem in understanding what 'distribution' actually means.